controlplane/kubeadm/internal/control_plane.go

```diff
@@ -385,3 +389,81 @@ func (c *ControlPlane) InjectTestManagementCluster(managementCluster ManagementC
 	c.managementCluster = managementCluster
 	c.workloadCluster = nil
 }
+
+// StatusToLogKeyAndValues returns following key/value pairs describing the overall status of the control plane:
+// - machines is the list of KCP machines; each machine might have additional notes surfacing
+//   - if the machine has been created in the current reconcile
+//   - if machines node ref is not yet set
+//   - if the machine has been marked for remediation
+//   - if there are unhealthy control plane component on the machine
+//   - if the machine has a deletion timestamp/has been deleted in the current reconcile
+//   - if the machine is not up to date with the KCP spec
+//
```

Comment:
```suggestion
```

Reply:
Goland suggest to add those lines after nested list, I think it doesn't hurt.

Reply:
This likely comes from gofmt and thus is enforced by Go